packages/myst-cli/src/session/session.ts

```diff
@@ -134,6 +148,7 @@ export class Session implements ISession {
   publicPath(): string {
     return path.join(this.sitePath(), 'public');
   }
+
   _clones: ISession[] = [];
 
   clone(): Session {
```

Comment:
We might need to put some logic into the clone session here, we don't want to be spanning a lot of jupyter servers to handle the results.

The session is cloned for each export, which goes through different transformation pipelines, but shouldn't create a new jupyter server.

Reply:
I've resolved this, but it's not pretty. As previously discussed, I'm just avoiding putting anything in the public API that we don't desperately need